av1an-core/src/scenes.rs

```diff
@@ -336,6 +336,23 @@ fn validate_zones_args() {
     .contains(&"--aq-mode=1".to_owned()));
 }
 
+#[test]
+fn validate_rav1e_zone_with_photon_noise() {
+  let input = "45 729 rav1e --speed 6 --photon-noise 4";
```

Comment:
I'm not an expert, but the tests seem to suggest "rav1e reset" instead of just "rav1e"?